### PR TITLE
sync-diff: add description for the new parameter `skip-non-existing-table`

### PR DESCRIPTION
**Pull request by liumengya94**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Add a description for the new parameter `skip-non-existing-table`, which is used to skip validation for tables that do not exist upstream or downstream.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):https://github.com/pingcap/tidb-tools/pull/693

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by lichunzhu — APPROVED** [review]
LGTM

File: sync-diff-inspector/sync-diff-inspector-overview.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ sync-diff-inspector 需要获取表结构信息、查询数据，需要的数据
 
 sync-diff-inspector 的配置总共分为五个部分：
 
-- Global config: 通用配置，包括校验的线程数量、是否输出修复 SQL 、是否比对数据等。
+- Global config: 通用配置，包括校验的线程数量、是否输出修复 SQL 、是否比对数据、是否跳过校验上游或下游不存在的表等。
```
Comment (Oreoxmt) [inline sync-diff-inspector/sync-diff-inspector-overview.md:62]:
```suggestion
- Global config: 通用配置，包括校验的线程数量、是否输出修复 SQL、是否比对数据、是否跳过校验上游或下游不存在的表等。
```

File: sync-diff-inspector/sync-diff-inspector-overview.md [hunk 2]
```diff
@@ -251,18 +255,21 @@ sync-diff-inspector 会在运行时定期（间隔 10s）输出校验进度到 c
...
+| `sbtest`.`sbtest96` | succeed | true               | +0/-101        |  999999 |   1000100 |
+| `sbtest`.`sbtest97` | skipped | false              | +999999/-0     |  999999 |         0 |
++---------------------+---------+--------------------+----------------+---------+-----------+
 Time Cost: 16.75370462s
 Average Speed: 113.277149MB/s
 ```
 
 - TABLE: 该列表示对应的数据库及表名
 
+- RESULT: 校验是否完成。若用户设置 `skip-non-existing-table = true`，对于上游或下游不存在的表，该列为`skipped`
```
Comment (Oreoxmt) [inline sync-diff-inspector/sync-diff-inspector-overview.md:271]:
```suggestion
- RESULT: 校验是否完成。若用户设置 `skip-non-existing-table = true`，对于上游或下游不存在的表，该列为 `skipped`
```

File: sync-diff-inspector/sync-diff-inspector-overview.md [hunk 3]
```diff
@@ -251,18 +255,21 @@ sync-diff-inspector 会在运行时定期（间隔 10s）输出校验进度到 c
...
-| `sbtest`.`sbtest96` | true               | +0/-101        |  999999 |   1000100 |
-+---------------------+--------------------+----------------+---------+-----------+
++---------------------+---------+--------------------+----------------+---------+-----------+
+|        TABLE        | RESULT  | STRUCTURE EQUALITY | DATA DIFF ROWS | UPCOUNT | DOWNCOUNT |
++---------------------+---------+--------------------+----------------+---------+-----------+
+| `sbtest`.`sbtest99` | succeed | true               | +97/-97        |  999999 |    999999 |
+| `sbtest`.`sbtest96` | succeed | true               | +0/-101        |  999999 |   1000100 |
+| `sbtest`.`sbtest97` | skipped | false              | +999999/-0     |  999999 |         0 |
++---------------------+---------+--------------------+----------------+---------+-----------+
 Time Cost: 16.75370462s
```
Comment (shichun-0415) [inline sync-diff-inspector/sync-diff-inspector-overview.md:265]: time cost 和 average speed 需要更新吗？

File: sync-diff-inspector/sync-diff-inspector-overview.md [hunk 4]
```diff
@@ -251,18 +255,21 @@ sync-diff-inspector 会在运行时定期（间隔 10s）输出校验进度到 c
...
 Average Speed: 113.277149MB/s
 ```
 
 - TABLE: 该列表示对应的数据库及表名
 
+- RESULT: 校验是否完成。若用户设置 `skip-non-existing-table = true`，对于上游或下游不存在的表，该列为 `skipped`
```
Comment (shichun-0415) [inline sync-diff-inspector/sync-diff-inspector-overview.md:271]:
```suggestion
- RESULT: 校验是否完成。如果你设置了 `skip-non-existing-table = true`，对于上游或下游不存在的表，该列为 `skipped`
```

File: sync-diff-inspector/sync-diff-inspector-overview.md (hunk 3 above)
Comment (liumengya94) [reply sync-diff-inspector/sync-diff-inspector-overview.md:265]: 不用哈

File: sync-diff-inspector/sync-diff-inspector-overview.md [hunk 5]
```diff
@@ -251,21 +255,21 @@ sync-diff-inspector 会在运行时定期（间隔 10s）输出校验进度到 c
...
 
-- TABLE: 该列表示对应的数据库及表名
-
-- STRUCTURE EQUALITY: 表结构是否相同
-
-- DATA DIFF ROWS: 即 `rowAdd` / `rowDelete`，表示该表修复需要增加/删除的行数
+- `TABLE`：该列表示对应的数据库及表名
+- `RESULT`：校验是否完成。如果设置了 `skip-non-existing-table = true`，对于上游或下游不存在的表，该列的值将为 `skipped`
+- `STRUCTURE EQUALITY`：表结构是否相同
+- `DATA DIFF ROWS`：即 `rowAdd` / `rowDelete`，表示该表修复需要增加/删除的行数
```
Comment (Oreoxmt) [inline sync-diff-inspector/sync-diff-inspector-overview.md:272]:
```suggestion
- `DATA DIFF ROWS`：即 `rowAdd`/`rowDelete`，表示该表修复需要增加/删除的行数
```
image

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Review by Oreoxmt — APPROVED** [review]

**Review by shichun-0415 — APPROVED** [review]

File: sync-diff-inspector/sync-diff-inspector-overview.md [hunk 6]
```diff
@@ -213,7 +215,9 @@ The data of `sbtest`.`sbtest99` is not equal
 The data of `sbtest`.`sbtest96` is not equal
 
 The rest of tables are all equal.
-The patch file has been generated in 
+
+A total of 2 tables have been compared, 0 tables finished, 2 tables failed, 0 tables skipped.
+The patch file has been generated in
         'output/fix-on-tidb2/'
 You can view the comparision details through 'output/sync_diff.log'
```
Comment (Oreoxmt) [inline sync-diff-inspector/sync-diff-inspector-overview.md:222]:
```suggestion
You can view the comparison details through 'output/sync_diff.log'
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Pull request merged**